.env.ci

```
@@ -0,0 +1,3 @@
+GITHUB_REPOSITORY=k3rnels-actions/pr-update
+TEST_PR_UTIL_EXPECTED_PR_NUMBER=6
+TEST_BODY_UTILS_EXPECTED_PR_LINK=[add: "test-feature" to "staging"](https://github.com/k3rnels-actions/pr-update/pull/174)
```

@k3rnelpan1c-dev do you think we would need single quotes for this ENV? 
A quick test on my bash actually fails with a `syntax error near unexpected token `('`

```suggestion
TEST_BODY_UTILS_EXPECTED_PR_LINK='[add: "test-feature" to "staging"](https://github.com/k3rnels-actions/pr-update/pull/174)'
```

---

to be on the safe side, lets do that